internal/logging/unix.go

```diff
@@ -0,0 +1,18 @@
+// +build !windows
```

Comment:
We've leaned towards using the term `posix` to encompass `linux`, `unix` and `darwin`. Thereby this filename should be `posix.go`.